addons/project/models/project.py

```diff
@@ -1368,6 +1369,10 @@ def _get_view(self, view_id=None, view_type='form', **options):
                 node.set('invisible', '1')
         return arch, view
 
+    @api.onchange('stage_id', 'project_id')
+    def _onchange_kanban_state(self):
+        self.kanban_state = 'normal'
```

Comment:
Why did you choose to do a onchange method instead of a compute method? 🤔

Reply:
as i know and verified by testing, with compute the value will be always 'normal' as compute is called at the load.

Reply:
🤔 I agree with @xavierbol that if the stage_id or project_id is changed via code then then the kanban_state won't be reset.

Reply:
If you make kanban_state = fields.Selection(..., compute="_compute...", readonly=False, store=True) it should work I think.

Reply:
yes it's working, thanks. i just added `compute` before and that's why the call was done at each load